src/test/test-fileio.c

```diff
@@ -612,6 +612,13 @@ static void test_tempfn(void) {
 
 static const char buffer[] =
         "Some test data\n"
```

Comment:
It seems we don't have a test for EOF meaning end of line. Can we add that here too?

Reply:
Would be nice.

Reply:
Added in #11160. PTAL